template/README.md

```diff
@@ -0,0 +1,38 @@
+---
+creation_date: 2024-05-29
+issues:
+- https://github.com/giantswarm/giantswarm/issues/30631
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+summary: This contains the RFC template for submitting a technical decision proposal
+---
+
+# This is a template that can be copied to create an RFC for submitting a technical decision proposal
+
+### Title
+<!-- Provide a concise and descriptive title for the RFC -->
+
+### Problem statement
+<!-- Explain the issue or challenge that needs to be addressed. This should include background information and context to help stakeholders understand why this decision is important. -->
+
+### Decision maker
+<!-- Identify the person (preferred) or a group responsible for making the final decision. -->
+
+### Deadline
+<!-- Specify the deadline by which the decision needs to be made. If there is no agreement or insufficient feedback before the deadline, the decision owner schedules a formal meeting with relevant stakeholders to discuss the RFC and make a decision -->
+
+### Who is affected / stakeholders
+<!-- List the individuals, teams, or SIGs that will be impacted by this decision and must provide feedback. -->
+
+### Preferred solution
+<!-- Describe the solution that is currently favored based on the analysis of the problem. -->
+
+### Alternative solutions
+<!-- Outline other potential solutions that were considered. For each alternative, provide a brief description and explain why it was not chosen as the preferred solution. -->
+
+### Implementation Plan
```

Comment:
The capitalisation here upsets my inner monk - is it specifically different here?

Reply:
no reason, just me becoming more German

Reply:
you've got to fight the change

Reply:
this isn't prufungsvorbereitung